c/src/alleleCounter.c

```diff
@@ -86,10 +92,11 @@ void alleleCounter_setup_options(int argc, char *argv[]){
              	{"min-base-qual", required_argument, 0, 'm'},
 							{"min-map-qual", required_argument, 0, 'q'},
 							{"is-snp6", required_argument, 0, 's'},
+							{"is-10x", required_argument, 0, 'x'},
 							{"contig", required_argument, 0, 'c'},
 							{"dense-snps", no_argument, 0, 'd'},
-							{"required-flag", no_argument, 0, 'f'},
-							{"filtered-flag", no_argument, 0, 'F'},
+							{"required-flag", optional_argument, 0, 'f'},
+							{"filtered-flag", optional_argument, 0, 'F'},
```

Comment:
`optional_argument` should be `required_argument` . The naming is misleading. But optional_argument actually means that the argument can optionally have a value. These both require a value to be given if the argument is set [docced here](https://linux.die.net/man/3/getopt_long)

Reply:
@drjsanger Ready for re-review